meta-flex-distro/conf/templates/default/local.conf.sample

```diff
@@ -14,26 +14,14 @@ MACHINE ??= "qemux86"
 # The EXTRA_IMAGE_FEATURES variable allows groups of packages to be added to
 # the generated images. See the Flex OS User’s and Reference Manual for
 # supported image features
-EXTRA_IMAGE_FEATURES ?= ""
-
-# Image features for development-image
-IMAGE_FEATURES_DEVELOPMENT ?= "debug-tweaks"
-IMAGE_FEATURES_DEVELOPMENT:append:flex-bsp = " codebench-debug ssh-server-openssh"
-EXTRA_IMAGE_FEATURES = "${IMAGE_FEATURES_DEVELOPMENT}"
+EXTRA_IMAGE_FEATURES ?= "debug-tweaks codebench-debug ssh-server-openssh"
```

Comment:
Hmm, I remember discussing to move this to development image recipe in our call. Weak assignment would ensure that users can add their own if they want.

Reply:
Missed the part of moving it to development-image. Updated now.